swagger-ext/src/test/scala/org/scalatra/swagger/SwaggerCommandSupportSpec.scala

```diff
@@ -38,7 +38,6 @@ object SwaggerCommandSupportSpec {
     get("/new", endpoint("new"), operation(newOperation)) { "OK" }
 
     protected def applicationDescription: String = "The command support servlet"
-
```

Comment:
whitespace consideration here too. I would avoid any cosmetic changes as much as possible. not just in hopes o merging upstream but to ease our efforts of keeping our fork up to date